examples/with-grafbase/.env.example

```diff
@@ -0,0 +1,2 @@
+GRAFBASE_API_URL=http://localhost:4000/graphql
+GRAFBASE_API_KEY=
```

Comment:
Assuming this is a secret, `.env.local.example` is more appropriate for a file name.